packages/teams-js/test/public/sharing.spec.ts

```diff
@@ -694,4 +696,104 @@ describe('sharing_v2', () => {
         }
       });
   });
+
+  describe('Testing getContent', () => {
```

Comment:
>describe [](http://example.com/codeflow?start=2&length=8)

Can you please add two tests:
- one that ensures that an error is thrown if the authorid is not a valid uuid
- one that ensures that an error is thrown if the threadid is not a valid uuid

Reply:
Updated tests for validating valid uuid,